### ISSUE TITLE
Tier system to deter hackers, reward long-term Nexus players

### ISSUE THREAD

**Issue by O0oRAVENo0O** [TheNexusMC-LLC/Issues#70]
Many hackers use multiple alternate accounts. This allows them to quickly rejoin the server and ban evade, even moments after being banned. 

This could be prevented by creating a tier system with possibly 2 or 3 levels using the level system already in place. Only lower the lowest tier would be accessible to new Nexus players (i.e. < level 10), and higher levels would gain access to the higher tier(s). This means that the vast majority of hackers and ban evaders would be kept in the lowest tier. 

In this way, the impact by hackers and gameplay would be minimized and dedicated Nexus players would gain a better playing experience. Additionally, the tier system could help improve competitiveness since it would allow higher-leveled players to match up against other experienced players.

**Comment by TheDracu** [comment]
Tier system as in what, more severe punishments?

**Comment by O0oRAVENo0O** [comment]
No. Each 'tier' would be a set of servers. For example, Tier 1 might be 20 SG servers available to all players. Tier 2 might be 20 servers accessible only to players above level 10. It would be unlikely that a  shameless hacker would reach level 10 before being banned, so Tier 2 would be mostly free of hackers. Thus, if a Nexus player over level 10 (which most devoted Nexus players are) wanted a good chance at a hacker-free game they could join a Tier 2 server.

**Comment by Deamon5550** [comment]
I'm not a fan of this as it forces new players to run a gauntlet of such when first joining.

**Comment by O0oRAVENo0O** [comment]
That's a fair criticism. However, the hackers are there regardless. This idea allows for a mostly hacker-free zone to exist and, further, as a reward for players who dedicate lots of time to the Nexus, encouraging them to play more. One last thing to consider is that the tier system only slightly increases the amount of hackers a new player would encounter. For example, if two hackers were on the Nexus with ~70 servers, the chance of encountering them would be ~2.9-percent. If the server was divided into two 35 server tiers that would rise to 5.7-percent, while the other tier would be hacker-free.

One last thing to consider is that the current system allows for hackers with (nearly) infinite alts to continue to return to the server, even after a moderator has put in the work to record, analyze, and ban them. The hacker will keep returning until they get bored of playing and leave on their own. Is this much better than never banning them at all and waiting for them to leave?

Anyway, that's the end of my explanation. Up to you if you want to implement it.

**Comment by Grqpes** [comment]
This is a good idea, it will make nexus players below level 10 want to play nexus more and get to level 10, and it makes sence, because most doners and people who really like the nexus would not risk hacking as this is they're favorite server and they want to keep playing it, please implement this system, or also maybe a server for just doners, as a very large precentile of donators are clean and don't hack!

**Comment by andy1ne0** [comment]
I'm iffy on this idea. While it would help reduce hackers and improve the experience for long-time players, it also means that newer players are exposed to a different level of competition. As such, the stats in games such as survival games become inconsistent (it'd be easier to move up on the kills leaderboard by only playing tier 1 games, even once past the first tier). For this reason in particular, this might not be the most feasible system. However, there's also the issue of player retention - if a new player only has access to, say, half the servers, they won't be as motivated to stay longer (particularly when they realise the higher-tier servers only have harder players). While it certainly gives new players something to work towards, the difference between the tiers wouldn't be significant enough to warrant the extra playing time, and as such new players would quickly become bored. Plus, level 10 does take a bit of time to achieve - level 4-5 might be more feasible, if at all.
